resources/aws/bucket.go

```diff
@@ -46,47 +44,25 @@ func (b *Bucket) CreateOrFail() error {
 }
 
 func (b *Bucket) Delete() error {
```

Comment:
For my interest. What is the difference in the implementations about buckets and bucket objects?